Examples/BasicTriangle.c

```diff
@@ -39,14 +39,7 @@ static int Init(Context* context)
 			.color_target_descriptions = (SDL_GPUColorTargetDescription[]){{
 				.format = SDL_GetGPUSwapchainTextureFormat(context->Device, context->Window),
 				.blend_state = {
```

Comment:
This isn't needed, right? Blend state should be getting zero-initialized from the designated initializer.